lib/flux-core/src/iter/range.rs

```diff
@@ -0,0 +1,60 @@
+use core::{iter::Step, ops};
```

Comment:
We still need to fix this such that it works for non-flux builds. We should either put this import under a `cfg` or the entire module.

Reply:
I put the module under a `cfg(flux)`